includes/class-scheduler.php

```diff
@@ -156,27 +156,28 @@ public static function update_followers() {
 		 *
 		 * @param int $number The number of followers to update.
 		 */
-		$number    = apply_filters( 'activitypub_update_followers_number', $number );
-		$followers = Followers::get_outdated_followers( $number );
+		$number = apply_filters( 'activitypub_update_followers_number', $number );
```

Comment:
Should we deprecate this filter in favor of `activitypub_update_remote_actors_number`?